content/en/docs/contributing/release-process.md

```diff
@@ -183,19 +261,33 @@ The process for cutting a minor release is as follows:
 
 6. Run `cmrel publish`
 
-   1. First do a dry-run, to ensure that all the staged resources are valid.
+   1. Find the "release name". The release name can be found in the output of
+      the previous command, `cmrel stage`. In the previous command output, look
+      for a line that looks like this:
 
-        ```bash
-        cmrel publish --release-name <RELEASE_NAME>
+        ```sh
```

Comment:
this is `text` now, not `sh`

Reply:
Done in 8f9273fd6c45b3819f7563cf7245da5e02a504ae! thanks!